src/services/api/features/package-actions/queries/issue-rai-query.ts

```diff
@@ -0,0 +1,9 @@
+export type IssueRaiQueryParams = {
+  id: string;
+  requestedDate: number;
+};
+
+export const issueRaiQuery = ({ id, requestedDate }: IssueRaiQueryParams) => `
```

Comment:
@pkim-gswell had a way of splitting things out in another pr that isolated types and objects in their respective files. Not sure if it makes sense to do here, but it was a nice pattern to keep functional files lighter. Thoughts, Paul?

Reply:
I would love to hear more. What you got @pkim-gswell ?

Reply:
In light of my rants against premature abstractions, Im of the opinion of avoiding SQL query function traps.  Instead I would raise this up to its usage:

```js
async issueRai(raiData: z.infer<typeof raiIssueSchema>) {
  await this.handleTransaction(async () => {
    await this.seatoolTransaction.request().query(`
      INSERT INTO SEA.dbo.RAI (ID_Number, RAI_Requested_Date)
      VALUES ('${raiData.id}',dateadd(s, convert(int, left(${raiData.requestedDate}, 10)), cast   ('19700101' as datetime)))
    `
    );
  });
}
```

Reply:
I know we had the idea of having specific queries be useful extracted _if_ we ran into the need to vary our endpoints for stuff like package actions and submissions. But, so far with CHIP SPA, none of that was really necessary. 🤔 

Thoughts @13bfrancis ? I'm indifferent, I'm but a follower of your conventions on that side of the app!